src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/EditorContextScope.cs

```diff
@@ -71,26 +72,40 @@ IIntellisenseResult IPowerFxScope.Suggest(string expression, int cursorPosition)
 
         public void AddQuickFixHandler(ICodeFixHandler codeFixHandler)
         {
-            _handlers.Add(codeFixHandler ?? throw new ArgumentNullException(nameof(codeFixHandler)));
+            if (codeFixHandler == null)
+            {
+                throw new ArgumentNullException(nameof(codeFixHandler));
+            }
+
+            _handlers.Add(codeFixHandler.GetType().Name, codeFixHandler);
```

Comment:
>codeFixHandler.GetType().Name [](http://example.com/codeflow?start=26&length=29)

How do we ensure that this key is consistent with the key used for lookup (on line 105)?

Reply:
+1